src/BuildScriptGenerator/BaseBashBuildScript.sh.tpl

```diff
@@ -46,7 +46,9 @@ then
 	{{ for excludedDir in DirectoriesToExcludeFromCopyToIntermediateDir }}
 	excludedDirectories+=" --exclude {{ excludedDir }}"
 	{{ end }}
-	rsync --delete -rt $excludedDirectories . "$INTERMEDIATE_DIR"
+
+	rsync --delete -rc $excludedDirectories . "$INTERMEDIATE_DIR"
```

Comment:
do we need to use check sum here as well? Can't we use timestamps since this is inside the container?

Reply:
yeah, we could probably, but just thought it will be consistent across. I will change it to use timestamp instead